### DISCUSSION TITLE
Blacklisted claums inside location tag

### DISCUSSION THREAD

**Discussion by Sugavanam** [OpenIDC/mod_auth_openidc#1110] [General]
Hello Team
    I have a conf file in apache with multiple application protected inside location tag.
Is it possible to use oidcblacklistedclaims inside location tag so that it is separate for each application ?

<Location /abc>
...
oidcblacklistedclaims secId lastname 
ProxyPass  https://www.abc.com
</Location >

<Location /xyz>
....
oidcblacklistedclaims  lastname givenname
ProxyPass  https://www.xyz.com
</Location >

Is there Any other way we can achieve this scenario?

**Comment by zandbelt** [comment]
`OIDCBlackListedClaims` works globally on the claims stored in the session, but you can either:
1. use e.g. `RequestHeader unset OIDC_CLAIM_givenname` to remove the claims you want to blacklist from the propagated headers
2. use `OIDCPassClaimsAs environment` with specific `RequestHeader set OIDC_CLAIM_sub "%{OIDC_CLAIM_sub}e"` primitives to pass only the claims that you want to in headers on a specific path
3. use `OIDCUserInfoClaimsExpr` when compiled with `libjq`, see: https://github.com/OpenIDC/mod_auth_openidc/blob/v2.4.14.3/auth_openidc.conf#L819-L835

**Comment by Sugavanam** [comment]
Thanks requestheader unset will solve my problem
